src/main/app/pdf/mappers/claimMapper.ts

```diff
@@ -9,7 +9,7 @@ export class ClaimMapper {
     const data = {
       submittedDate: MomentFormatter.formatLongDateAndTime(claim.createdAt),
       claimNumber: claim.claimNumber,
-      amount: NumberFormatter.formatMoney(claim.claimData.amount),
+      amount: NumberFormatter.formatMoney(claim.claimData.amount.totalAmount()),
```

Comment:
revert

Reply:
If we keep the amount as breakdown in caimData then we need this change

Reply:
I would really prefer to keep amount as just a value, its much nicer to work with.
We have no need of the amount breakdown in the frontend currently